templates/setenv.sh.erb

```diff
@@ -46,3 +36,83 @@ cd $PUSHED_DIR
 
 echo ""
 echo "Server startup logs are located in $LOGBASEABS/logs/catalina.out"
+<%- if version[0] >= 6 and version[1] >= 1 # version 6.1 or greater -%>
```

Comment:
I would have used versioncmp() but this approach is also fine.

Reply:
That was the function (method?) I was trying to remember. :)

I thought it was something maybe built into the String type, didn't think to look there (Puppet::Util::Package).